featuretools/primitives/standard/aggregation_primitives/__init__.py

```diff
@@ -0,0 +1,57 @@
+# flake8: noqa
```

Comment:
I'm pretty sure this line can be deleted in every __init__.py file. I added an ignore to these files in `.flake8`

Reply:
We can't delete it in the files that use the `from ... import *` approach or we get this linting error:

```
featuretools/primitives/standard/__init__.py:4:1: F403 'from featuretools.primitives.standard.transform_primitives import *' used; unable to detect undefined names
```

I was able to remove it from a couple other places though.